### PR TITLE
Update zhCN.lua

### PR DESCRIPTION
**Pull request by nanjuekaien1**
L.DisplaySourceDebugInfo = "Display Source, ID and Debug Info in tooltip."
I'm not localizing for now, and I haven't figured out what it's for. . .

### CODE REVIEW

**Comment by Xruptor** [comment]
> L.DisplaySourceDebugInfo = "Display Source, ID and Debug Info in tooltip." I'm not localizing for now, and I haven't figured out what it's for. . .

That option displays debugging information within the tooltip to make things easier for me to help others debug in the future.

What it does is display the **source** (meaning what function or location) the error is coming from, the **ID** is things like ItemID, CurrencyID, etc... and **Debug** Info is additional information associated with the item displaying the BagSync tooltip information.

Basically it's just an option to show Debug information.  Perhaps I should relabel it to make it clearer.

In fact let me do just that :)  I'll close this merge and then you can make a new one with the updated changes.
